components/SockJsOrderClient.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react'
+import {apiRoot} from "../constants/Backend";
+import SockJsClient from 'react-stomp';
+import * as Device from 'expo-device';
+
+
+const url = `${apiRoot}/ws`
+const debugMode = false
+
+
+class RealTimeOrderUpdateBase extends Component {
+
+  constructor(props, context) {
+    super(props, context)
+  }
+
+  render() {
+    const {
+      meta,
+      topics,
+      handleOnMessage,
+      id,
+    } = this.props;
+
+    return (
+      <SockJsClient url={url} topics={[topics]}
+        onMessage={(data) => {
+          if (data === `${id}.order.orderChanged`) {
+            handleOnMessage()
+            console.log(`get device: ${Device.deviceName}-${Device.modelName}`)
+          }
+          else if (data === `${id}.inflightOrders.ordersChanged`) {
+            handleOnMessage()
+            console.log(`refresh tables: ${Device.deviceName}-${Device.modelName}`)
+          } else {
+            handleOnMessage()
+            console.log(`refresh orders: ${Device.deviceName}-${Device.modelName}`)
+          }
```

Comment:
Is it possible to move the if else check to the onMessage implementation to further simplify this component?